### ISSUE TITLE
Is sip.linhome.org down?

### ISSUE THREAD

**Issue by pgorod** [BelledonneCommunications/linhome-android#39]
Some 3 or 4 days ago, some working configurations I had with Linhome stopped working. It seems that sip.linhome.org is not responding, is that possible?

Symptoms: door system trying to connect to sip.linhome.org gets stuck "connecting...". I didn't change any configurations, and this was working before...

I thought it was just for me but I heard from a friend that also uses linhome that it's broken for him too.

Any debugging I can do? Anyone else with this issue?

**Comment by KyronNextLevel** [comment]
Hello.
Belledonne Communications' sysadmin here.

TLDR : Linhome service should be back online today.

I was in charge of a non-planned Linhome migration last week.
We will likely communicate on more channels later but to be short :
An unplanned migration of the Linhome infrastructure had to be done, because a planned migration of the subscribe.linphone.org affected also subscribe.linhome.org while it shouldn't have been the case...
I have re-deployed the Linhome infrastructure but forgot to test the server connectivity over TCP/UDP on port 5060.
Linhome apps were registering ok, because they are using port 5061 my default.

My apologize for the inconvenience.

**Comment by pgorod** [comment]
I confirm it's working again! Thanks for the honest clarification, it is appreciated. 

And thanks for the Linhome service!

**Comment by pgorod** [comment]
This is acting very erratic again, lately. Most of the times the calls don't work - is there any other maintenance going on? 

If there's any logging or troubleshooting that I can do on my side to help solve the problems, just ask. Thanks

**Comment by pgorod** [comment]
This issue is happening again... No sip connection for a couple of days... :-(

**Comment by pgorod** [comment]
I'm afraid this is happening once more... @KyronNextLevel can you please fix this? Thank you

**Comment by pgorod** [comment]
And once again sip.linhome.org seems to be down... :-(

@KyronNextLevel if you can please take a look?

I don't know if my messages here when the system is down are useful or not. Also, if there is a page somewhere to monitor service status please let me know. 

Although I only talk to you guys when your service is letting me down, still **I am thankful** for all the days when your (free!) service is working great.

**Comment by BelledonneComm** [comment]
Hello @pgorod.
We had to switch our infrastructure because of an unplanned maintenance.
Some configuration keys were missing for Linhome on the other server.
Service is back online.

Sorry for the inconvenience.

**Comment by KyronNextLevel** [comment]
Hello @pgorod.
We had to switch our infrastructure because of an unplanned maintenance.
Some configuration keys were missing for Linhome on the other server.
Service is back online.

Sorry for the inconvenience.
